src/collections/utils.ts

```diff
@@ -68,28 +68,54 @@ function getBaseAPIPath(pathFromUrl?: string | null) {
   return baseApiPath === '/' ? '' : baseApiPath.substring(1, baseApiPath.length - 1);
 }
 
+export enum AstraEnvironment {
```

Comment:
I have mixed feelings about exposing the idea of other non-prod environments here. I feel it could introduce confusion. This is just a utility right? Maybe we just assume prod.

Reply:
This is an optional parameter to the `createAstraUri` function which defaults to `PRODUCTION`.

So the typical usage of `createAstraUri(dbIdUUID,'us-east1','testks1', 'myToken')` is supported and optionally we can pass the environment as below
`createAstraUri(dbIdUUID,'us-east1','testks1', 'myToken', AstraEnvironment.PRODUCTION)`

Reply:
Kathir, here the enum definition will fail the switch statement in createAstraUri method.
May change to 
export enum AstraEnvironment {
  PRODUCTION = "PRODUCTION",
  TEST = "TEST",
  DEVELOPMENT = "DEVELOPMENT"
}

Reply:
@Yuqi-Du Do you see an issue? Can you please post the error you are seeing? In the switch statement, I'm using the variable `astraEnv?: AstraEnvironment` which is not a `string`, so should be fine I guess.

Reply:
@jeffreyscarpenter BTW, thinking more on the option itself, I agree with you on the `environment` option in general, since this is not an option for any public user. Let me remove this option and update.